### PR TITLE
Create SingleInheritence

### PR DESCRIPTION
**Pull request by Kavimaha123**
Hi.. I have completed today task in SingleInheritence.Please check as same like small doubt in here program...

### CODE REVIEW

File: SingleInheritence/SingleInheritence [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+public class Main {
+    public static void main(String[] args) {
+        // Create an instance of class B
+        B childObj = new B("user123", "password123");
+
+        // Test validation
+        String inputUserName = "user123";
+        String inputPassword = "password123";
+
+        boolean isValid = childObj.validate(inputUserName, inputPassword);
```
Comment (vijayVijeyender) [inline SingleInheritence/SingleInheritence]: this variable is not necessary , you can directly pass the function inside if condition
if(childObj.validate(inputUserName,InputPassword)

File: SingleInheritence/SingleInheritence [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        this.password = password;
+    }
+
+    // Method to validate username and password
+    boolean validate(String userName, String password) {
+        return this.userName.equals(userName) && this.password.equals(password);
+    }
+}
+
+// Class B - Child
```
Comment (vijayVijeyender) [inline SingleInheritence/SingleInheritence]: create this class in a new file  => B.java

File: SingleInheritence/SingleInheritence [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// Class A - Parent
+class A {
```
Comment (vijayVijeyender) [inline SingleInheritence/SingleInheritence]: Rename this file to =>from SingleInheritance to  A.java

**Comment by vijayVijeyender** [comment]
Good Work so far

**Comment by Kavimaha123** [comment]
Create new pull request 3

**Pull request closed without merging**